test/integration/shell_int_test.go

```diff
@@ -349,7 +348,6 @@ func (suite *ShellIntegrationTestSuite) TestPs1() {
 
 	cp = ts.SpawnWithOpts(
 		e2e.OptArgs("shell", "small-python"),
-		e2e.OptAppendEnv(constants.DisableRuntime+"=false"),
```

Comment:
Doesn't `state shell` require a runtime having been setup? I'm surprised this does not fail.

Reply:
No, `shell` will still happily create a subshell without a runtime.

Reply:
I see. I didn't know that.